frontend/stylesheets/trace_view/_user_settings_view.less

```diff
@@ -5,3 +5,7 @@
 .centered-children {
   text-align: center;
 }
+
+.tracing-settings-menu .ant-tabs-content {
+  padding-right: 8px;
```

Comment:
I added some padding to the settings on the left to not get elements covered by the scrolling bar.